packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics-batcher.ts

```diff
@@ -441,7 +441,7 @@ describe('plugin-metrics', () => {
         // item also gets assigned a delay property but the key is a Symbol and haven't been able to test that..
         assert.deepEqual(calls.args[0].eventPayload, {
           event: 'my.event',
-          origin: {buildType: 'test', networkType: 'unknown'},
+          origin: {buildType: 'test', networkType: 'unknown', upgradeChannel: 'test'},
```

Comment:
Can we also have a test for `prod` cases here?

Reply:
Hello @mkesavan13, thank you for the comment. This is already tested [here](https://github.com/webex/webex-js-sdk/blob/2a2c7d894455736247497febabd6e2beed3e7be5/packages/%40webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.util.ts#L449).